validator/src/cli.rs

```diff
@@ -881,6 +884,14 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .validator(is_parsable::<usize>)
                 .help("Controls the TPU connection pool size per remote address"),
         )
+        .arg(
+            Arg::with_name("tpu_max_connections_per_ipaddr_per_minute")
+                .long("tpu-max-connections-per-ipaddr-per-minute")
+                .takes_value(true)
+                .default_value(&default_args.tpu_max_connections_per_ipaddr_per_minute)
+                .validator(is_parsable::<u32>)
+                .help("Controls the rate of the clients connections per IpAddr per minute."),
```

Comment:
are we prepared to offer guidance as to when/if/how this should be tuned? if not, lets hide it until we are

Reply:
Put to hidden unless forced